README.md

```
@@ -52,11 +57,83 @@
-End-to-end tests (Playwright) can be run against a running development environment. See the Playwright configuration (`apps/web/playwright.config.ts`) and test files (`apps/web/e2e/`) for details.
+## End-to-End and Performance Testing
 
-Performance tests (Locust) can be run against the running API service. See `locustfile.py` for the test definition.
+These advanced tests require a running application environment and are recommended to be executed as part of a pre-release checklist or before merging significant changes to the main branch.
+
+### Playwright End-to-End Tests
+
+Playwright tests are used for end-to-end testing of the frontend application.
+
+**Prerequisites:**
+*   The full application (both `web` and `api` services) must be running. You can start them using:
```

markdownlint-cli2 (0.17.2)

Lists should be surrounded by blank lines
null

(MD032, blanks-around-lists)